package.json

```diff
@@ -71,15 +73,22 @@
     "eslint-plugin-unicorn": "~44.0.0",
     "eslint-plugin-vue": "~9.5.1",
     "eslint-plugin-vue-pug": "~0.5.4",
-    "esno": "0.16.3",
     "json-schema": "~0.4.0",
     "json-schema-to-typescript": "~11.0.2",
     "prettier": "2.7.1",
     "prettier-plugin-organize-imports": "~3.1.1",
+    "ts-dedent": "~2.2.0",
+    "tsx": "~3.9.0",
     "typescript": "~4.8.4",
     "upper-case-first": "~2.0.2",
+    "vitest": "~0.23.4",
     "vue-eslint-parser": "~9.1.0"
   },
+  "pnpm": {
+    "overrides": {
+      "@poppinss/colors": "poppinss/colors#fix/typing-nbuild"
+    }
+  },
```

Comment:
Please let's wait until the PR is merged, because I don't want dependencies that are unversioned and can always change

Reply:
Done !